xlators/mgmt/glusterd/src/glusterd-bitrot.c

```diff
@@ -48,11 +48,12 @@ __glusterd_handle_bitrot(rpcsvc_request_t *req)
     char msg[256] = {
         0,
     };
-    xlator_t *this = THIS;
+    xlator_t *this = NULL;
     glusterd_conf_t *conf = NULL;
 
     GF_ASSERT(req);
-
+    this = req->trans->xl;
```

Comment:
For this function the use of 'this' is for 2 reasons:
1. To get 'conf' - which seems to only be used for 'conf->op_version' comparisons, which hopefully I'll be able to eliminate via https://github.com/gluster/glusterfs/pull/3544
2. For error messages, for which case we can use 'THIS', I believe.